tools/project_manager/external_catalog_search.php

```diff
@@ -108,16 +194,18 @@ function do_search_and_show_hits()
     $id = yaz_connect($external_catalog_locator, ["charset" => "UTF-8"]);
     yaz_syntax($id, "usmarc");
     yaz_element($id, "F");
-    yaz_search($id, "rpn", trim($fullquery));
+    yaz_search($id, "rpn", trim(str_replace("\n", " ", $fullquery)));
     $extra_options = ["timeout" => 60];
     yaz_wait($extra_options);
     $errorMsg = yaz_error($id);
 
+    echo "<details style='border: 1px solid #aaa; border-radius: 4px; padding: 0.5em 0.5em 0'>\n";
```

Comment:
Should we move some of this styling into CSS? I'd rather have labeled-snowflakes there than style dusted around the code.

Reply:
Done, after a smaller adventure in learning about `lessc`.